inc/metadata/namespace.php

```diff
@@ -1219,19 +1220,95 @@ function check_thema_lang_file( $post ) {
 
 }
 
+/**
+ * Transform the institution list to better use in the book info page
+ *
+ * @param array $institutions
+ *
+ * @return array
+ */
+function transform_institutions( array $institutions ): array {
+	return array_reduce( $institutions, static function( $carry, $institution ) {
+		return array_merge( $carry, [ $institution['code'] => $institution['name'] ] );
+	}, [] );
+}
```

Comment:
Do you think this method should be tested? I'm testing only `get_institutions` and `get_institution_by_code` for now.